.eslintrc.json

```diff
@@ -0,0 +1,109 @@
+{
+	"root": true,
+	"extends": [
+		"eslint:recommended",
+		"plugin:@typescript-eslint/recommended",
+		"plugin:import/errors",
+		"plugin:import/typescript"
+	],
+	"env": {
+		"es6": true
+	},
+	"plugins": [
+		"prettier",
+		"import"
+	],
+	"parserOptions": {
+		"ecmaVersion": 6,
+		"sourceType": "module"
+	},
+	"parser": "@typescript-eslint/parser",
+	"settings": {
+		"import/parsers": {
+			"@typescript-eslint/parser": [".ts"]
+		},
+		"import/resolver": {
+			"typescript": {
+				"alwaysTryTypes": true,
+				"directory": "./tsconfig.json"
+			}
+		}
+	},
+	"rules": {
+		"import/order": ["error", {
+			"groups": [
+				"builtin",
+				"external",
+				"internal",
+				"parent",
+				"sibling",
+				"index"
+			],
+			"pathGroups": [{
+				"pattern": "config/**",
+				"group": "internal"
+			}, {
+				"pattern": "parse/**",
+				"group": "internal"
+			}, {
+				"pattern": "shared/**",
+				"group": "internal"
+			}, {
+				"pattern": "types/**",
+				"group": "internal"
+			}, {
+				"pattern": "utils/**",
+				"group": "internal"
+			}],
+			"newlines-between": "always",
+			"alphabetize": {
+				"order": "asc",
+				"caseInsensitive": true
+			}
+		}],
```

Comment:
HI @evs-chris, I've added `eslint-plugin-import` to consistently handle imports. Are you ok with the following configuration?

Reply:
Yep, that looks fine to me.